src/client/pages/OfferNew/Checkout/InsuranceSummary.tsx

```diff
@@ -197,7 +197,8 @@ const getDetails = (
           label: textKeys.CHECKOUT_DETAILS_QUOTE_TYPE(),
           value:
             quoteDetails.__typename === 'SwedishApartmentQuoteDetails' ||
-            quoteDetails.__typename === 'NorwegianHomeContentsDetails'
+            quoteDetails.__typename === 'NorwegianHomeContentsDetails' ||
+            quoteDetails.__typename === 'DanishHomeContentsDetails'
```

Comment:
I get your point. I added `quoteDetails.__typename === 'DanishHomeContentsDetails'` here since the `'NorwegianHomeContentsDetails'` was already here and Danish home contents should work as the Norwegian one. I think this is a good thing to take into account when we refactor this but for now I feel like it's more important to get this to work so we can get the entire embark->offer flow and un-block the app team.

Reply:
Or maybe I'm misunderstanding.

Reply:
Yeah as you said, its was already like that for NorwayHomeContent. I it was more of a general question but I
 think I might have been a bit unclear what i meant. 
We are using the CHECKOUT_APARTMENT textkey for when we display HomeContentDetails. So my question was if don't have a specifik text key for HomeContentDetails but rather that we use CHECKOUT_APARTMENT for HomeContent as well.

Reply:
Aha, I see! Yes maybe we should change the text key, you have a point. Not gonna do that now though.